### ISSUE TITLE
Is it possible to control node artifact location / repo structure?

### ISSUE THREAD

**Issue by amnonkhen** [srs/gradle-node-plugin#245]

**Comment by srs** [comment]
No, sorry, that is not possible. But, are you using a corporate repo for caching?

**Comment by amnonkhen** [comment]
sorry for the silence period. 
Yes I am using a corporate repo.
The format is ${url}/npm-repo/mode-[version].tgz
How can I have the plugin download from there?
